django_elasticsearch_dsl/management/commands/search_index.py

```diff
@@ -116,7 +116,7 @@ def _populate(self, models, options):
         parallel = options['parallel']
         for doc in registry.get_documents(models):
             self.stdout.write("Indexing {} '{}' objects {}".format(
-                doc().get_queryset().count() if options['count'] else "all",
+                doc().get_indexing_queryset().count() if options['count'] else "all",
```

Comment:
As `get_indexing_queryset()` returns a `generator` object, I think we won't be able to invoke `count()` on it.